extensions/wikia/AdEngine/js/provider/yavliTag.js

```diff
@@ -1,18 +1,18 @@
 /*global define*/
 define('ext.wikia.adEngine.provider.yavliTag', [
```

Comment:
What do you think about removing whole yavli? Worth to ask Nef/Nandy and make some small clean up :)